modelexpress_client/python/tests/test_vmm_arena.py

```
@@ -0,0 +1,668 @@
+
+    def test_first_malloc_at_base(self):
+        arena = VmmArena(total_bytes=1 << 30)
+        a = arena.malloc(1024)
+        assert a == arena.base
+
+    def test_size_rounds_up_to_granularity(self):
+        backend = _StubBackend()
+        arena = VmmArena(total_bytes=1 << 30, backend=backend)
+        arena.malloc(1024)
+        # Backend sees the granularity-rounded size, not the raw request.
+        va, size, handle = backend.allocations[0]
```

Ruff (0.15.12)

[warning] Unpacked variable `va` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)